swap/Cargo.toml

```diff
@@ -64,6 +63,13 @@ url = { version = "2", features = ["serde"] }
 uuid = { version = "0.8", features = ["serde", "v4"] }
 void = "1"
 
+[dependencies.torut]
+branch = "feature-flag-tor-secret-keys"
+default-features = false
+features = ["v3", "control"]
+git = "https://github.com/bonomat/torut/"
+version = "0.1"
```

Comment:
We temporarily depend on this fork until the PR is merged: https://github.com/teawithsand/torut/pull/10

Reply:
You might be interested in learning about the [`[patch]`](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section) section inside `Cargo.toml` files. Useful to do things like this without having to modify the original manifest.

The good thing is that `[patch]` is applied across the whole dependency tree and hence also affects transitive dependencies.

Reply:
cool stuff :)